src/schema/sources.ts

```diff
@@ -575,6 +586,25 @@ export const resolvers: IResolvers<any, Context> = {
         throw err;
       }
     },
+    removeSource: async (
+      _,
+      { sourceId }: { sourceId: string },
+      ctx,
+    ): Promise<GQLEmptyResponse> => {
+      const sourceMember = await ctx.con
+        .getRepository(SourceMember)
+        .findOneByOrFail({ sourceId, userId: ctx.userId });
+      if (sourceMember.role !== SourceMemberRoles.Owner) {
+        throw new ForbiddenError(
+          'Access denied! You do not have permission for this action!',
+        );
+      }
+      await ctx.con.getRepository(Source).delete({
+        id: sourceId,
+        active: false,
```

Comment:
Why do you check for active false?

Reply:
Because at time of writing you could only delete inactive sources. (1 member)
With the new addition this is not needed.